src/js_command_match/add_key.rs

```diff
@@ -3,49 +3,116 @@
 pub struct AddKeyArgs {
     account_id: String,
     access_key: String,
-    #[clap(long, aliases = ["contract_id", "contractId"], default_value = None)]
+    #[clap(long, aliases = ["contractId"], default_value = None)]
```

Comment:
I vaguely remember that JS CLI actually supported three ways of passing flags: camelCase (`--contractId`), kebab-case (`--contract-id`), and snake_case (`--contract_id`). Are you sure that we don't need to support snake_case?